### DISCUSSION TITLE
[ToggleGroup] : How do I make it work with a form?

### DISCUSSION THREAD

**Discussion by AbePlays** [radix-ui/primitives#2654] [Help]
Upon submitting a form containing a ToggleGroup, the selected values do not appear in the FormData. While using an input type=checkbox resolves the issue but I believe there might be concerns about its correctness in terms of a11y.

**Comment by benoitgrelard** [comment]
`ToggleGroup` wasn't designed to work with forms so I believe this will be an issue. You could use `RadioGroup` instead?
